pyartcd/pyartcd/pipelines/update_golang.py

```diff
@@ -186,33 +182,42 @@ async def run(self):
             jenkins.init_jenkins()
             jenkins.update_title(title_update)
 
-        # Do a sanity check for rhel8 module build
-        # a module build should never be tagged directly in override tag
-        # if it is, we should make sure it is available via module inheritance
-        # and then untag it from override tag
-        if 8 in el_nvr_map and 'module' in el_nvr_map[8]:
-            tag = f'rhaos-{self.ocp_version}-rhel-8-override'
-            package = parse_nvr(el_nvr_map[8])['name']
-            nvr = get_latest_nvr_in_tag(tag, package, self.koji_session)
-            if nvr:
-                raise ValueError(f'{nvr} is tagged in {tag}, please make sure it is available via module inheritance '
-                                 f'(brew list-tagged {tag} {package} --inherit). If not, use --create-tagging-ticket. '
-                                 f'Once it is available, untag it from the override tag (brew untag-build {tag} {nvr})')
-
-        # if requested, create ticket for tagging request
-        if self.create_ticket:
-            _LOGGER.info('Making sure that builds are not already tagged')
-            for el_v, nvr in el_nvr_map.items():
-                if is_latest_build(self.ocp_version, el_v, nvr, self.koji_session):
-                    raise ValueError(f'{nvr} is already the latest build, run '
-                                     'only with nvrs that are not latest or run without --create-tagging-ticket')
-
-            _LOGGER.info('Creating Jira ticket to tag golang builds in buildroots')
-            self.create_jira_ticket_for_el8(el_nvr_map, go_version)
```

Comment:
can you remove this func and all the flags/params related to self.create_ticket

Reply:
updated